modules/snooker/src/main/ts/ephox/snooker/api/Structs.ts

```diff
@@ -1,4 +1,5 @@
 import { SugarElement } from '@ephox/sugar';
+import { Arr } from '@ephox/katamari';
```

Comment:
Nitpick: import is out of order, `k` is before `s`, so this should be above the sugar import.